regression/abstraction/abstraction.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
If the directory is renamed `abstraction.sh` would be less of a problem but, again, it would be better with a less ambigious name.

Reply:
Ok, will rename abstraction --> rr-abstraction.

Reply:
Changed it to rr-abstraction.sh.